lib/galaxy/dataset_collections/structure.py

```diff
@@ -180,16 +180,12 @@ def tool_output_to_structure(get_sliced_input_collection_type, tool_output, coll
         collection_type_descriptions = collections_manager.collection_type_descriptions
```

Comment:
`collection_type_descriptions` is now unused and makes flake8 complain.

Reply:
I've rebased the PR with the fix.